package-lock.json

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
Could we add this file to the `.gitignore`? I don't think it should be committed

Reply:
Done (see latest commit)